src/views/BookList.vue

```diff
@@ -0,0 +1,50 @@
+<template lang="pug">
```

Comment:
I am a little confuse with this, solo home page / only have navbar y you don't use ?